<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectMainWindow</class>
 <widget class="QMainWindow" name="NewProjectMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openStacksButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Stacks</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="messages">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>471</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>38</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>tif stack filename common </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filenameCommon">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>36</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>475</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
